package org.connect.fitconnect.core.database

import org.connect.fitconnect.core.Error

enum class DataError : Error {
    UNKNOWN, DATA_NOT_FOUND
}